pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,196 @@
+//+build linux
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connections
+
+import (
+	"net"
+	"os"
+
+	"github.com/ti-mo/conntrack"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/util/sysctl"
+)
+
+var _ ConnTrackDumper = new(connTrackDumper)
+
+type connTrackDumper struct {
```

Comment:
I am a bit surprised we don't test that code in integrations test. Was it intentional?

Reply:
Sounds good, please remember to do it